src/install.c

```diff
@@ -630,6 +630,13 @@ static gboolean launch_and_wait_default_handler(RaucInstallArgs *args, gchar* bu
 		mfimage = l->data;
 		dest_slot = g_hash_table_lookup(target_group, mfimage->slotclass);
 
+		if (dest_slot->external_mount_point && !r_umount(dest_slot->device, &ierror)) {
```

Comment:
It might not work calling unmount with the device path rather than the actual mount point. At least busybox mount could not handle this properly when I tested it for the last time.